<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weather</class>
 <widget class="QWidget" name="weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>553</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1677215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
    <pointsize>10</pointsize>
    <weight>10</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
</string>
  </property>
  <widget class="QLabel" name="label_nowtime">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>当前时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>100</width>
     <height>81</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="weather_picture.qrc">:/new/prefix1/天气/中雨.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>391</width>
     <height>266</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="city">
      <property name="text">
       <string>城市</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="now_weather">
      <property name="text">
       <string>当前天气</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nowT">
      <property name="text">
       <string>现在温度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="air">
      <property name="text">
       <string>空</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Cur_banner">
      <property name="text">
       <string>提示标语</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="remind">
      <property name="text">
       <string>提醒</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>城市</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_city">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255,0);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_search">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>751</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_time1">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_date1">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_picture1">
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>图片</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_weather1">
        <property name="text">
         <string>天气</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_highT1">
        <property name="text">
         <string>高温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lowT1">
        <property name="text">
         <string>低温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind1">
        <property name="text">
         <string>风的名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind_scale1">
        <property name="text">
         <string>风等级</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_time2">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_date2">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_picture2">
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>图片</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_weather2">
        <property name="text">
         <string>天气</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_highT2">
        <property name="text">
         <string>高温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lowT2">
        <property name="text">
         <string>低温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind2">
        <property name="text">
         <string>风的名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind_scale2">
        <property name="text">
         <string>风等级</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_time3">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_date3">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_picture3">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>图片</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_weather3">
        <property name="text">
         <string>天气</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_highT3">
        <property name="text">
         <string>高温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lowT3">
        <property name="text">
         <string>低温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind3">
        <property name="text">
         <string>风的名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind_scale3">
        <property name="text">
         <string>风等级</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_time4">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_date4">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_picture4">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>图片</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_weather4">
        <property name="text">
         <string>天气</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_highT4">
        <property name="text">
         <string>高温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lowT4">
        <property name="text">
         <string>低温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind4">
        <property name="text">
         <string>风的名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind_scale4">
        <property name="text">
         <string>风等级</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_time5">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_date5">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_picture5">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>图片</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_weather5">
        <property name="text">
         <string>天气</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_highT5">
        <property name="text">
         <string>高温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lowT5">
        <property name="text">
         <string>低温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind5">
        <property name="text">
         <string>风的名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind_scale5">
        <property name="text">
         <string>风等级</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_time6">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_date6">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_picture6">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>图片</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_weather6">
        <property name="text">
         <string>天气</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_highT6">
        <property name="text">
         <string>高温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lowT6">
        <property name="text">
         <string>低温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind6">
        <property name="text">
         <string>风的名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind_scale6">
        <property name="text">
         <string>风等级</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_time7">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_date7">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_picture7">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>图片</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_weather7">
        <property name="text">
         <string>天气</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_highT7">
        <property name="text">
         <string>高温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lowT7">
        <property name="text">
         <string>低温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind7">
        <property name="text">
         <string>风的名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wind_scale7">
        <property name="text">
         <string>风等级</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_return1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="weather_picture.qrc"/>
 </resources>
 <connections/>
</ui>
